docs/Review_Apps.md

```diff
@@ -1,131 +1,135 @@
-## Working with review apps
+# Working with review apps
 
-Hokusai provides a command for managing review apps. Review apps are useful for testing feature branches that are not yet ready to be deployed to staging but need to be tested in a staging-like environment.
+Hokusai provides commands for managing review apps. Review apps are useful for testing features that are not yet ready to be deployed to staging but need to be tested in a staging-like environment.
 
-In order to start a review app you will need to follow these steps:
+Perform the following steps in order to create a review app. `app-name` refers to name of the review app.
 
-1) Create a new review app
-    ```shell
-    hokusai review_app setup <name> # we recommend using branch name or PR number as name
-    ```
-    This command will create a new `<name>.yml` under `hokusai/` folder.
+## Prepare app
 
-2) Check the newly created `<name>.yml` file and make sure everything looks good. Note that we use Kubernetes [`namespace`](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) for review apps. Basically each review app will end up being in its own namespace to not collide with staging.
+```
+hokusai review_app setup <app-name> # we recommend using branch name or PR number as name
+```
 
-3) Push an image with this review app tag:
+It creates a `<app-name>.yml` file under `hokusai/` folder. This is the review app Yaml.
 
-    ```shell
-    hokusai registry push --tag <name>
-    ```
+Check the file and make sure everything looks good. Note that we use Kubernetes [`namespace`](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) for review apps, that is, each review app is created in a dedicated namespace.
 
-    If you have git-ignored files in your working directory (likely) you will have to force push with:
-    ```shell
-    hokusai registry push --force --tag <name>
-    ```
+## Push Docker image
 
-    It may be a good idea to skip updating the `latest` tag in the registry, which you can do with:
-    ```shell
-    hokusai registry push --skip-latest --tag <name>
-    ```
+Tag the image with review app's name.
 
-4) Make sure your review app deployment will use the image you just pushed. This can be done by modifying the new `<name>.yml` file.
+```
+hokusai registry push --tag <app-name>
+```
 
-    There will be a `containers` subsection of the configuration that specifies the `image` that should be pulled from AWS Elastic Container Registry when booting up your review app. Update the value of that `image` so that it points to your newly tagged image, rather than the default staging image for that project.
+If there are git-ignored files in the working directory, which is likely, force push:
 
-    Example:
-    ```yml
-    image: <aws-account-id>.dkr.ecr.us-east-1.amazonaws.com/volt:staging
+```
+hokusai registry push --force --tag <app-name>
```

Comment:
So `--force` will include the git ignored files?

Reply:
yeah

Reply:
wait, let me check.

Reply:
[git-ignored files should also be docker-ignored.](https://github.com/artsy/hokusai/pull/428/commits/42c7abc062cc30adeb840ca265003ffd56b4d562)